ui.apps/src/main/content/jcr_root/apps/core/cif/clientlibs/common/js/PriceFormatter.js

```diff
@@ -17,11 +17,80 @@
 
 class PriceFormatter {
     constructor(locale) {
-        this._locale = locale;
+        this._locale = locale || window.CIF.locale || navigator.language;
```

Comment:
You can also use `document.documentElement.lang` as a fallback. That's the setting coming from the AEM page properties.